### PR TITLE
Fetch built docs and extract

### PR DESCRIPTION
**Pull request by Sutrannu**
Signed-off-by: Sutrannu <bbetts@bitwise.io>

### CODE REVIEW

File: generator/entrypoint-web.sh [hunk 1]
```diff
@@ -2,19 +2,74 @@
...
+    sed -i '/versions.json/s/https.*json/\/docs\/versions.json/' ./archive/docs/*/*/*/_static/version_switcher.js
+    cp -rd ./archive/docs/* /usr/local/apache2/htdocs/docs/
     cp -rf /srv/jekyll/_site/* /usr/local/apache2/htdocs/
     rm /srv/jekyll/*.complete
-
     /usr/local/apache2/bin/httpd -DFOREGROUND
 else
     rm /srv/jekyll/*.complete
-	echo "Site was not built">&2
+	info "Site was not built"
```
Comment (vaporos) [inline generator/entrypoint-web.sh]: Odd intent here, unintentional tab?

File: docker-compose.yaml [hunk 2]
```diff
@@ -31,4 +31,4 @@ services:
     ports:
       - 8000:80
     volumes:
-      - $PWD/generator:/srv/jekyll
\ No newline at end of file
+      - $PWD/generator:/srv/jekyll
```
Comment (vaporos) [inline docker-compose.yaml]: This newline-only change should probably be a separate commit.

Comment (vaporos) [reply docker-compose.yaml]: Same for other line-only changes.

**Review by rberg2 — APPROVED** [review]
I was able to test this today and it looks good to me.

**Comment by rberg2** [comment]
Can we get the complete website or webroot copied out of the docker container and into the src directory? maybe some place  like `sawtooth-website/build` or similar.

**Comment by Sutrannu** [comment]
> Can we get the complete website or webroot copied out of the docker container and into the src directory? maybe some place like `sawtooth-website/build` or similar.
@rberg2 
The web root will be in `generator/archive/htdocs` after build.

**Comment by Sutrannu** [comment]
> Move `docs/index.html` into the site builder

**Review by dcmiddle — APPROVED** [review]

**Review by vaporos — APPROVED** [review]
